package.json

```diff
@@ -59,6 +59,7 @@
   },
   "devDependencies": {
     "@types/node": "^16.11.7",
+    "@types/tmp": "^0.2.3",
```

Comment:
Was `tmp` not used in any other place we just had it `dependencies`?

Reply:
Sorry but can you please be more clear about it?

Reply:
We have `tmp` in the `dependencies` in `package.json`. I suppose the typing package is needed if it is actually used within Typescript. I wonder why this one was not needed so far, which could mean that we do not actually use the package prior to this change.

Reply:
This is actually quite funny lol.